### PR TITLE
Enable in cluster wait for aggregator jobs

### PR DESCRIPTION
**Pull request by xueqzhan**
[TRT-1240](https://issues.redhat.com/browse/TRT-1240)

/hold for now

### CODE REVIEW

**Comment by xueqzhan** [comment]
/payload-aggregate periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-sdn-upgrade 2

File: ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh [hunk 1]
```diff
@@ -4,11 +4,13 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+export KUBECONFIG=/var/run/kubeconfig/kubeconfig
 job-run-aggregator analyze-job-runs \
   --google-service-account-credential-file=${GOOGLE_SA_CREDENTIAL_FILE} \
   --job=${VERIFICATION_JOB_NAME} \
   --aggregation-id=${AGGREGATION_ID} \
   --explicit-gcs-prefix=${EXPLICIT_GCS_PREFIX} \
   --job-start-time=${JOB_START_TIME} \
```
Comment (DennisPeriquet) [inline ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh:13]: does this need `-query-source=cluster `?

**Comment by DennisPeriquet** [comment]
I'm holding and lgtm'ing in case my question is answered and everything can stay the same.  Please `/hold cancel` if we're all set.

File: ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh (hunk 1 above)
Comment (xueqzhan) [reply ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/openshift-release-analysis-prpqr-aggregator-commands.sh:13]: Yup, it is two lines below.

**Pull request merged**